src/test/ui/borrowck/borrowck-unary-move.rs

```diff
@@ -1,3 +1,6 @@
+// revisions: ast mir
+//[mir]compile-flags: -Z borrowck=mir
```

Comment:
Is this change intended?

Reply:
Whoops, no, I must have messed up the rebase.